test/pkg/test/framework.go

```diff
@@ -86,11 +88,8 @@ func NewFramework(ctx context.Context) *Framework {
 		TestCasesCreatedServiceNames:        make(map[string]bool),
 		TestCasesCreatedTargetGroupNames:    make(map[string]bool),
 	}
-
-	SetDefaultEventuallyTimeout(180 * time.Second)
+	SetDefaultEventuallyTimeout(5 * time.Minute)
```

Comment:
not following, I dont know why it was 3 minutes before and now it should be 5, some magic numbers